sdk/storage/azure-storage-blob-stress/src/main/java/com/azure/storage/blob/stress/DownloadStream.java

```diff
@@ -35,21 +32,18 @@ public DownloadStream(StorageStressOptions options) {
     }
 
     @Override
-    protected boolean runInternal(Context span) {
-        ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
-        try {
+    protected boolean runInternal(Context span) throws IOException {
+        try (CrcOutputStream outputStream = new CrcOutputStream()) {
             syncClient.downloadStreamWithResponse(outputStream, null, null, null, false, null, span);
-            return ORIGINAL_CONTENT.checkMatch(BinaryData.fromBytes(outputStream.toByteArray()), span).block().booleanValue();
-        } catch (Exception e) {
-            LOGGER.error("Failed to download blob", e);
-            return false;
+            outputStream.close();
```

Comment:
Any reason for closing here while we're using a try-with-resource block which will close this after we leave the scope?

Reply:
The `close` is the only way to tell we're done writing to this stream, and that crc/length/data is all there - we'll need it to be over in the `ORIGINAL_CONTENT.checkMatch`